packages/fetchai/protocols/consensus/README.md

```diff
@@ -0,0 +1,42 @@
+# Consensus Protocol
+
+## Description
+
+This is a consensus protocol for aggregating observations.
+
+## Specification
+
+```yaml
+---
+name: consensus
+author: fetchai
+version: 0.1.0
+description: A protocol for agents to aggregate individual observations
+license: Apache-2.0
+aea_version: '>=0.9.0, <0.10.0'
+speech_acts:
+  observation:
+    value: pt:int
+    time: pt:int
+    source: pt:str
+    signature: pt:str
+  aggregation:
+    value: pt:int
+    time: pt:int
+    contributors: pt:list[pt:int]
+    signature: pt:str
+...
+---
+initiation: [observation, aggregation]
+reply:
+  observation: []
+  aggregation: []
+termination: [observation, aggregation]
+roles: {agent}
+end_states: [successful, failed]
```

Comment:
There seems to be only one state the dialogue ends in (only one message).

Reply:
Actually, with the current setup, there are two outcomes, depending on which move is used (ending in observation or aggregation)